### PR TITLE
Fix a bug of transfering float to int by mistake

### PR DESCRIPTION
**Pull request by shadow-blade-X**

### CODE REVIEW

File: PlateUp-flask/run.py [hunk 1]
```diff
@@ -255,7 +255,9 @@ def __filter_by_time(self, recipe_list, filter_time_h, filter_time_min):
     def __filter_recipe(self, recipe_list, filter_cost, filter_time_h, filter_time_min, filter_has_ingredient):
         if len(recipe_list)==0:
             self.random_pick=True
-            recipe_list=db.session.query(Recipe).all() # return all BUG
+            #recipe_list=self.__search_for_recipes_by_tags("popular") # return popular recipes
```
Comment (eanile) [inline PlateUp-flask/run.py]: Get rid of commented code if not needed.

File: PlateUp-flask/run.py [hunk 2]
```diff
@@ -377,8 +379,10 @@ def get(self):
         if self.random_pick:
             recipe_list = random.sample(recipe_list, k=min(len(recipe_list), int(limit)))
             page = 0
-        
+
```
Comment (eanile) [inline PlateUp-flask/run.py:381]: Remove extra newline

File: PlateUp-flask/run.py (hunk 1 above)
Comment (eanile) [reply PlateUp-flask/run.py]: Also why was the comment return all BUG before? Is that a bug?

File: PlateUp-flask/run.py [hunk 3]
```diff
@@ -255,7 +255,6 @@ def __filter_by_time(self, recipe_list, filter_time_h, filter_time_min):
     def __filter_recipe(self, recipe_list, filter_cost, filter_time_h, filter_time_min, filter_has_ingredient):
         if len(recipe_list)==0:
             self.random_pick=True
-            #recipe_list=self.__search_for_recipes_by_tags("popular") # return popular recipes
             recipe_list=db.session.query(Recipe).all()
```
Comment (eanile) [inline PlateUp-flask/run.py:258]: Still curious as to why in a previous version this line had BUG next to it

Comment (shadow-blade-X) [reply PlateUp-flask/run.py:258]: Initially we want to return a random recipe, then we got the tag and we want the recipe with popular tag. But the popular recipe is broken then we are back to the origin.

**Review by eanile — APPROVED** [review]

**Pull request merged**